jobs/payment-jobs/tasks/cfs_create_account_task.py

```diff
@@ -119,6 +120,8 @@ def create_accounts(cls):  # pylint: disable=too-many-locals
             pending_account.save()
 
             # Publish message to the Queue, saying account has been created. Using the event spec.
+            payment_info.update(
```

Comment:
Not part of this ticket, but we will need to move this event publishing to pay-api. Job will be executed only when CFS is up, but we need to send the PAD agreements when the account is created, else the 3 day period won't work.

Reply:
sure.will handle in the next PR